projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -101,14 +104,17 @@
 	display: none;
 }
 
-.contact-form label.checkbox {
-	font-weight: 400;
+.contact-form :where(label.checkbox-multiple),
+.contact-form :where(label.radio),
+.contact-form :where(label.checkbox) {
+
```

Comment:
Tiniest of nits: unintended single line whitespace? (not a blocker)

Reply:
Unfortunately Jetpack's linter auto adds this above any comments (there's one on the following line).

Reply:
Ah, good to know! Ignore me then 🙂